packages/react-query/src/scripts/utils/table.ts

```diff
@@ -0,0 +1,50 @@
+import Table from 'cli-table3'
+import { getPackageJson, getTanStackReactQueryPackageJson } from './package'
+
+export const VERSION5_APIS = [
+  '<SuspenseQuery/>',
+  '<SuspenseQueries/>',
+  '<SuspenseInfiniteQuery/>',
+  '<Mutation/>',
+  '<QueryErrorBoundary/>',
+]
+
+export const VERSION4_APIS = [
+  'useSuspenseQuery',
+  'useSuspenseQueries',
+  'useSuspenseInfiniteQuery',
+  'queryOptions',
+  'infiniteQueryOptions',
+  '<SuspenseQuery/>',
+  '<SuspenseQueries/>',
+  '<SuspenseInfiniteQuery/>',
+  '<Mutation/>',
+  '<QueryErrorBoundary/>',
+]
+
+export function getStatusTable(currentVersion: string) {
+  const packageJson = getPackageJson()
+  const tanStackReactQueryPackageJson = getTanStackReactQueryPackageJson()
+  const tanStackReactQueryMajorVersion = tanStackReactQueryPackageJson.version.split('.')[0]
+
+  const table = new Table({
+    head: ['', 'result', 'status', 'advice'],
+    style: { head: [] },
+  })
+
+  console.log(tanStackReactQueryMajorVersion)
+
+  table.push(['version', `@suspensive/react-query@${packageJson.version}`, '🟢', ''])
+  table.push(['export', `@suspensive/react-query-${currentVersion}@${packageJson.version}`, '🟢', ''])
```

Comment:
```suggestion
  table.push(['export', `@suspensive/react-query-${currentVersion}@${targetPackageJson.version}`, '🟢', ''])
```

This should be not packageJson(@suspensive/react-query's package.json) targetPackageJson(@suspensive/react-query-4, @suspensive/react-query-5's package.json)

Reply:
I added `getSuspensiveReactQueryPackageJson` to accurately retrieve the targetVersion.

- [feat(react-query): add getSuspensiveReactQueryPackageJson util](https://github.com/toss/suspensive/pull/1100/commits/ce9d72a744bfec5dc0028b62399f7cbaf9299317)